src/main/ngapp/src/app/main/main.component.html

```diff
@@ -2,27 +2,27 @@
   <nb-layout-header fixed>
     <div class="header-container">
       <div class="text-center">Auto Translating Chat App</div>
-
-      <nb-actions size="medium" class="pull-right">
-        <nb-action
-          icon="person-outline"
-          status="primary"
-          [nbContextMenu]="menuItems"
-          nbContextMenuTag="my-context-menu"
-        ></nb-action>
-      </nb-actions>
+      <button nbButton size="large" class="pull-right" [nbContextMenu]="menuItems" nbContextMenuTag="my-context-menu">
+          <nb-icon icon="person-outline" status="primary" size="large"></nb-icon>
+      </button>
     </div>
   </nb-layout-header>
 
   <ng-template #dialog let-data let-ref="dialogRef">
-    <nb-card class="dialog-template">
-      <nb-card-header class="header-container" (click)="ref.close()">
+    <nb-card class="dialog-template" [nbSpinner]="languageUpdateLoading" nbSpinnerStatus="primary">
+      <nb-card-header class="header-container">
         Update Language Preference
-        <button nbSuffix nbButton ghost autofocus>
+        <button nbSuffix nbButton ghost autofocus (click)="(ref.close() && closeAlert())">
```

Comment:
will this close the pop up card automatically (one a user has successfully selected a language to change to)? 
i had noticed that the pop-up window stayed until a user clicked the 'x' close button even after language had been successfully changed.

Reply:
The window stays open until a user closes it with the x buttton. After the language has been updated, we just show the user the updated message